perfkitbenchmarker/analyze.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
```

Comment:
I think the package idea is a good idea but it's out of scope for this PR. We have an upcoming work item to refactor the object storage benchmark and as part of that work, we can consider this option. For now, all I am asking is to rename analyze.py to something more specific to object storage benchmark, e.g.: "object_storage_multistream_analysis.py", that's it. Then we can resolve this comment and move on from this PR.